packages/core/src/layout/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -28,15 +30,54 @@ type State = {
 
 type EProps = {
   error?: Error;
-  slackChannel?: string;
+  slackChannel?: typeof defaultSlackChannel;
   children?: React.ReactNode;
 };
 
-const Error = ({ slackChannel }: EProps) => {
+const Error = ({ slackChannel, error }: EProps) => {
+  const [isShowingTrace, setIsShowingTrace] = useState(false);
   return (
     <div role="alert">
-      Something went wrong here.{' '}
-      {slackChannel && <>Please contact {slackChannel} for help.</>}
+      <CardHeader title="Something Went Wrong" />
+      <CardContent>
+        <p>
+          <strong>Error:</strong> {error?.message ?? 'No Further Info'}
+        </p>
+        <div>
+          <Button
+            onClick={() => setIsShowingTrace(!isShowingTrace)}
+            variant="contained"
+            color="primary"
+          >
+            {isShowingTrace ? 'Hide' : 'Show'} Details
+          </Button>
+        </div>
+        {isShowingTrace && (
+          <pre style={{ overflow: 'auto' }}>
+            <code>
+              {error?.stack?.split('\n').map(line => (
+                <>
+                  {line}
+                  <br />
+                </>
+              )) ?? ''}
+            </code>
+          </pre>
+        )}
+        {slackChannel && (
+          <p>
+            Please contact{' '}
+            <a
+              href={slackChannel.href}
+              target="_blank"
+              rel="noopener noreferrer"
```

Comment:
Why add these tbh? Could probably be useful information to have

Reply:
Chrome seems to have changed the default in 88 though, and seeing and the process security model is a Chrome thing to begin with it should be fine to skip now. Including the referrer I'm just thinking could be good for tracking internally within a company be able to make informed decisions.

Don't wanna block this, but also not thinking this is a pattern that we need to be using too much.

Reply:
@freben figured that we where actually doing this internally too, so can easily settle the debate with that we could keep doing it at least for now. Speaking of which, could we use our own `Link` component here? That's where we'd want to collect these kind of things rather than having them spread out.